src/utils/createElement.js

```diff
@@ -1,11 +1,15 @@
+export function createElement(tag, attributes) {
```

Comment:
what if I want to create a button and add custom callback to it? I think it wont work